pkgs/by-name/mo/modprobed-db/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, libevdev
+, kmod
+, bash
+, installShellFiles
+}:
+let
```

Comment:
```suggestion
}:

let
```

Reply:
why?

Reply:
Staying consistent with other packages (plus it looks cleaner IMO)

Reply:
I disagree. This change makes no impact on the readibility on the derivation, and if ir was it should have been handled ny the formarter.